lib/vrl/tests/src/main.rs

```diff
@@ -17,7 +17,7 @@ use vrl::VrlRuntime;
 use vrl::{diagnostic::Formatter, state, Runtime, Terminate, Value};
 use vrl_tests::{docs, Test};
 
-#[cfg(not(target_env = "msvc"))]
+#[cfg(feature = "jemalloc")]
```

Comment:
I think the semantics here are slightly different now. If I run `cargo` in `lib/vrl/tests` it'll now use the system allocator by default on *nix where before it used `jemalloc`. It's not a big deal, I think we only had this flag in here when we ran into a weird bug with jemalloc that we never quite resolved, just worked around: https://github.com/vectordotdev/vector/issues/6726

Reply:
Though, maybe it's better to keep the tests running under `jemalloc` by default in case we hit more weird edge cases like that 😅

Reply:
Ah, right, the semantics are different. The `unix` feature doesn't properly get flagged on in this crate.